### ISSUE TITLE
enforce similarity loss, try

### ISSUE THREAD

**Issue by hasibzunair** [hasibzunair/masksup-segmentation#15]
```
import torch.nn.functional as F
loss = F.mse_loss(x, y)
```
